playbooks/ufw-playbook.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- hosts: frontend_servers
+  roles:
+    - franklinkim.ufw
```

Comment:
Uh, I might be thinking of the ansible `iptables` module: https://docs.ansible.com/ansible/iptables_module.html -- which I was looking at before looking at ufw.

I'll investigate!